fury/interactor.py

```diff
@@ -331,10 +401,17 @@ def _callback(_obj, event_name):
             # Update event information.
             interactor_ = self.GetInteractor()
             if interactor_ is not None:
-                self.event.update(event_name, interactor_)
+                # self.event.update(event_name, interactor_)
```

Comment:
We can delete this line since it was moved in `_process_event`

Reply:
okay :+1: